app/src/main/java/com/kickstarter/ui/activities/compose/search/FilterMenuBottomSheet.kt

```diff
@@ -0,0 +1,296 @@
+                }
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+private fun ProjectStatusRow(
+    modifier: Modifier = Modifier,
+    text: String = stringResource(R.string.Filter_fpo),
+    selectedStatus: MutableState<DiscoveryParams.PublicState?> = mutableStateOf(null),
+    callback: (DiscoveryParams.PublicState?) -> Unit = {},
+) {
+    val backgroundDisabledColor = colors.backgroundDisabled
+    val dimensions: KSDimensions = KSTheme.dimensions
+    var switchChecked by remember { mutableStateOf(false) }
+
+    val pillOptions = listOf(
+        null to stringResource(R.string.Project_Status_All_fpo),
+        DiscoveryParams.PublicState.LIVE to stringResource(R.string.Project_Status_Live_fpo),
+        DiscoveryParams.PublicState.LATE_PLEDGE to stringResource(R.string.Project_Status_Late_Pledges_fpo),
+        DiscoveryParams.PublicState.SUCCESSFUL to stringResource(R.string.Project_Status_Successful_fpo),
+        DiscoveryParams.PublicState.UPCOMING to stringResource(R.string.Project_Status_Upcoming_fpo),
+    )
+
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .drawBehind {
+                drawLine(
+                    color = backgroundDisabledColor,
+                    start = Offset(0f, size.height),
+                    end = Offset(size.width, size.height),
+                    strokeWidth = dimensions.dividerThickness.toPx()
+                )
+            }
+            .padding(
+                start = dimensions.paddingLarge,
+                top = dimensions.paddingLarge,
+                bottom = dimensions.paddingLarge,
+                end = dimensions.paddingMediumSmall
+            )
+    ) {
+        Column(verticalArrangement = Arrangement.spacedBy(KSTheme.dimensions.listItemSpacingSmall)) {
+            Text(
+                text = text,
+                style = typographyV2.headingLG,
+                color = colors.textPrimary
+            )
+
+            FlowRow(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.spacedBy(8.dp),
+                verticalArrangement = Arrangement.spacedBy(8.dp)
+            ) {
+                pillOptions.forEach { (state, label) ->
+                    Timber.d("*****${FilterMenuTestTags.pillTag(state)}")
+                    PillButton(
+                        text = label,
+                        shouldShowIcon = false,
+                        isSelected = selectedStatus.value == state,
+                        modifier = Modifier.testTag(FilterMenuTestTags.pillTag(state)),
+                        onClick = {
+                            val newSelection = if (selectedStatus.value == state) null else state
+                            selectedStatus.value = newSelection
+                            callback(newSelection)
+                        }
+                    )
+                }
+            }
+
+            Row(
```

Comment:
This is hidden UI for now, is the `Include ended projects` switch, will be shown once the query is able to take multiple `PublicProjectStates` and not just one.